doc/subport/sonic-sub-port-intf-hld.md

```diff
@@ -96,63 +102,98 @@ A sub port interface shall support the following features:
 * VRF
 * RIF counters
 * QoS setting inherited from parent physical port or port channel
-* mtu inherited from parent physical port or port channel
+* MTU: 
+  MTU inherited from parent physical port or port channel. 
+  If subinterface MTU is configured, MTU on subinterface will be configured with:
+  - If Subinterface MTU <= parent port MTU, configured subinterface MTU will be applied.
+  - If Subinterface MTU > parent port MTU, parent port MTU will be applied.
 * Per sub port interface admin status config
+  - Kernel subinterface netdev admin UP can be performed only if parent interface netdev is admin UP.
```

Comment:
Do we have any db-migrator section to describe for long subinterface name to short interface name conversion across releases?

Reply:
Yes, we have to handle subinterface name conversion in db_migrator script during upgrade & downgrade.
Updated in next commit.

Reply:
Updated to support both short name and long name format for creating subinterfaces. With this no need to handle DB conversion while performing upgrade.